client/extensions/wp-super-cache/cache-stats.jsx

```diff
@@ -2,49 +2,97 @@
  * External dependencies
  */
 import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
+import { flowRight, get } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 import FoldableCard from 'components/foldable-card';
+import { errorNotice, removeNotice } from 'state/notices/actions';
+import { deleteFile } from './state/stats/actions';
+import { hasFileDeleteError, isDeletingFile } from './state/stats/selectors';
 
-function getAge( { lower_age, upper_age } ) {
-	if ( lower_age && upper_age ) {
-		return `${ lower_age } - ${ upper_age }`;
-	} else if ( lower_age ) {
-		return lower_age;
-	} else if ( upper_age ) {
-		return upper_age;
+function getAge( lower, upper ) {
+	if ( lower && upper ) {
+		return `${ lower } - ${ upper }`;
+	} else if ( lower ) {
+		return lower;
+	} else if ( upper ) {
+		return upper;
 	}
 
 	return '';
 }
 
 class CacheStats extends Component {
 	static propTypes = {
+		deleteFile: PropTypes.func.isRequired,
+		errorNotice: PropTypes.func.isRequired,
 		files: PropTypes.array,
+		hasError: PropTypes.bool,
 		header: PropTypes.string,
+		isDeleting: PropTypes.bool,
+		removeNotice: PropTypes.func.isRequired,
+		siteId: PropTypes.number.isRequired,
 		translate: PropTypes.func.isRequired,
 	};
 
 	static defaultProps = {
 		files: [],
+		hasError: false,
+		header: '',
+		isDeleting: false,
```

Comment:
This seems ever so slightly code-smelly -- the selectors for `hasError` and `isDeleting` should return `false` by default IMO, so there shouldn't be a need to set default props for them.